examples/playground/index.html

```diff
@@ -42,8 +42,8 @@
       </div>
     </div>
 
-    <script src="env.ts"></script>
-    <script src="darkMode.ts"></script>
-    <script src="app.tsx"></script>
+    <script type="module" src="env.ts"></script>
+    <script type="module" src="darkMode.ts"></script>
+    <script type="module" src="app.tsx"></script>
```

Comment:
Should we handle this fix (and fix all examples) in a separate PR? We'll also need to add `/** @jsxRuntime classic */` pragma in some files.

Reply:
probably, I think it's semi-fixed through other PRs already, but happy to revert

Reply:
#1081, if it gets merged before this PR